src/rpc/rawtransaction.cpp

```diff
@@ -74,7 +74,6 @@ static UniValue getrawtransaction(const JSONRPCRequest& request)
             "is known, its hash can be provided even for nodes without -txindex. Note that if a blockhash is\n"
             "provided, only that block will be searched and if the transaction is in the mempool or other\n"
             "blocks, or if this node does not have the given block available, the transaction will not be found.\n"
-            "DEPRECATED: for now, it also works for transactions with unspent outputs.\n"
```

Comment:
> I'd start without blockhash argument (mempool only), then blockhash, then say blockhash can be omitted if txindex.

I don't think this captures the logic. "without blockhash argument (mempool only)" is not correct, since we'll check the whole blockchain if txindex is enabled. This also doesn't communicate that if blockhash is provided, then txindex won't be used to check the entire blockchain.